openedx.yaml

```diff
@@ -0,0 +1,9 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: adap
+oeps: {}
+openedx-release: {parent-repo: edx/edx-analytics-dashboard}
+owner: edx/analytics
+tags: [analytics]
```

Comment:
I'm not sure why `pyyaml` dumped the output in the inline format. If we figure out how to make it behave better, we could do a bulk PR to reformat them. Otherwise, I'm not going to bother.